### ISSUE TITLE
Version 0.7.0.20 error: "pull access denied for test1, repository does not exist"

### ISSUE THREAD

**Issue by steveendow** [microsoft/navcontainerhelper#1160]
It appears the error that was resolved in 0.7.0.19 (Issue #1153) has returned in version 0.7.0.20.

The error occurs if the named image does not exist.  The named image gets created successfully, but the error occurs when trying to create the container.  After the named image is created, running the script again creates the container successfully.

Full transcript is attached.

Error message:

```
D:\BCPowerShell\New BC Test Container with Elapsed Time_v1.ps1 : Error response from daemon: pull
access denied for test1, repository does not exist or may require 'docker login': denied:
requested access to the resource is denied
```

New-BcContainer Script:

```
    $containerName = "test"
    $imageName = $containerName + "1"
    $password = 'P@ssw0rd'
    $securePassword = ConvertTo-SecureString -String $password -AsPlainText -Force
    $credential = New-Object pscredential 'admin', $securePassword
    $auth = 'UserPassword'
    $artifactUrl = Get-BcArtifactUrl -type 'Sandbox' -version '' -country 'us' -select 'Latest'
    New-BcContainer `
        -accept_eula `
        -containerName $containerName `
        -credential $credential `
        -auth $auth `
        -artifactUrl $artifactUrl `
        -imageName $imageName `
        -dns $dnsIP `
        -updateHosts 
```

[2020-07-26 1110 test Container Build Log.txt](https://github.com/microsoft/navcontainerhelper/files/4978169/2020-07-26.1110.test.Container.Build.Log.txt)

**Comment by freddydk** [comment]
This happens if you have exactly one docker image downloaded before running new-bccontainer.
Will be fixed in next version

**Comment by freddydk** [comment]
fix shipped in 0.7.0.21
